catalog/task/run-file-updates/0.1/run-file-updates.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-file-updates
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to run file updates
+  params:
+    - name: jsonKey
+      type: string
+      description: The json key containing the file updates
+      default: ".spec.extraData.fileUpdates[]"
+    - name: fileUpdatesPath
```

Comment:
IIUC this means that we need another task (or adjust the `collect-data` one to store the fileUpdatesPath file, right?